src/uxnasm.c

```diff
@@ -380,13 +382,13 @@ build(char *rompath)
 		if(labels[i].name[0] - 'A' > 25 && !labels[i].refs)
 			printf("-- Unused label: %s\n", labels[i].name);
 	fwrite(data + PAGE, length - PAGE, 1, dst);
-	/* printf(
```

Comment:
I think we're intentionally silencing this for the purposes of the discord bot

cc @neauoire